video.py

```diff
@@ -0,0 +1,38 @@
+
+import requests
```

Comment:
Since this class and the Customer class didn't end up being used, it's generally better to delete stuff like this before submitting the PR. 
Not a big deal, just wanted to note for future reference!